### PR TITLE
upgrade sqlagg to 0.3.0

### PR DESCRIPTION
**Pull request by nickpell**
requires https://github.com/dimagi/sql-agg/pull/16

### CODE REVIEW

**Comment by nickpell** [comment]
sqlagg 0.3.0 not found in tests

**Comment by czue** [comment]
@nickpell to fix tests you'll have to wait for the dependent sqlagg PRs to get merged and then publish the new version to pypi. see https://confluence.dimagi.com/display/commcarehq/Python+Packaging+Crash+Course for more information (also happy to help with this if that page isn't clear)

File: corehq/apps/userreports/reports/filters.py [hunk 1]
```diff
@@ -54,23 +67,28 @@ def to_sql_values(self):
...
+        '<=': LTEFilter,
+        '<': LTFilter,
+    }
 
     def __init__(self, filter, value):
         assert filter.type == "numeric"
         assert (isinstance(value, dict) and "operator" in value and "operand" in value) or value is None
         if value:
-            assert value['operator'] in ["=", "!=", "<", "<=", ">", ">="]
+            assert value['operator'] in self.operators_to_filters.keys()
```
Comment (snopoke) [inline corehq/apps/userreports/reports/filters.py]: You need the `.keys()`. You can just say `assert value['operator'] in self.operators_to_filters`

Comment (czue) [inline corehq/apps/userreports/reports/filters.py]: (I think @snopoke meant "don't need")

Comment (snopoke) [inline corehq/apps/userreports/reports/filters.py]: 👍

On 29 October 2015 at 09:59, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/userreports/reports/filters.py
> <https://github.com/dimagi/commcare-hq/pull/8839#discussion_r43357681>:
>
> >
> >      def __init__(self, filter, value):
> >          assert filter.type == "numeric"
> >          assert (isinstance(value, dict) and "operator" in value and "operand" in value) or value is None
> >          if value:
> > -            assert value['operator'] in ["=", "!=", "<", "<=", ">", ">="]
> > +            assert value['operator'] in self.operators_to_filters.keys()
>
> (I think @snopoke <https://github.com/snopoke> meant "don't need")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8839/files#r43357681>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

Comment (nickpell) [inline corehq/apps/userreports/reports/filters.py]: cleaned up

**Comment by nickpell** [comment]
:white_check_mark:

**Pull request merged**